website/docs/sql-function-glossary/sql-aggregate-functions/sql-avg.md

```diff
@@ -0,0 +1,51 @@
+---
+id: sql-avg
+title: SQL AVG
+description: The AVG function is used to straightforwardly calculate the average of a numeric column, but you may also see it used in a window function to calculate rolling averages.
+slug: /sql-function-glossary/sql-avg
+---
+
+<head>
+    <title>Working with the SQL AVG function</title>
+</head>
+
+You’re a data person, so we assume you’re going to be calculating averages of some metrics \**waves hands airly*\* at some point in your career. And the way to calculate averages of a numeric column in SQL is by using the AVG function.
```

Comment:
Is "airly" a word?  Never seen that one, but I get it and like it

Reply:
meant to be airily!

Reply:
Fixed in commit